starter/src/app/app.component.html

```diff
@@ -49,7 +49,6 @@
 						</div>
 
 						<div class="stark-app-bar-page-title">
-							<h1>{{ name }}</h1>
```

Comment:
I guess this line should be kept in the Starter right? Because we will only show our logo in the Showcase... or am I missing something?